backend/controllers/error_controller.py

```diff
@@ -95,7 +96,8 @@ def get_first_same_date_indexes_in_sublists(dates1, dates2, index1, index2):
     return get_first_same_date_indexes_in_sublists(dates1, dates2, index1, index2 + 1)
 
 
-class Error(Resource):
+class Error(ProtectedResource):
+
     def get(self, filename1, filename2):
         data1, _, _ = get_data(os.path.join(config.UPLOAD_FOLDER, secure_filename(filename1)))
```

Comment:
Didn't you change the filename convention or does the client make the switch?

Reply:
In the file_controller I include the new filename in the response and that's what the client sends in this request, so yes the client makes the switch

Reply:
Such kind of subtle and non-obvious conventions should probably be documented some in the code - ideally both in client and server code where this needs to happen.